client/my-sites/site-settings/disconnect-site/pagination-flow.jsx

```diff
@@ -0,0 +1,44 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Gridicon from 'gridicons';
+import Main from 'components/main';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+
+class PaginationFlow extends Component {
```

Comment:
I'm curious about the component name, `PaginationFlow`, wouldn't it make more sense if we swap the word order,`FlowPagination` or just `Pagination` or even just `SkipSomething` - `SkipSurvey`. I don't see a real pagination except for a skip link, so maybe that would make sense?

Reply:
Yes, this definitely could use a better name 👍  `Skip Survey` it is!